src/components/common/Extensions/UserHub/partials/NotificationsTab/partials/Notification/Action/MultisigNotificationMessage.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useMemo, type FC } from 'react';
+import { defineMessages } from 'react-intl';
+
+import { getActionTitleValues } from '~common/ColonyActions/helpers/index.ts';
+import {
+  type ColonyActionFragment,
+  type NotificationColonyFragment,
+} from '~gql';
+import { useAmountLessFee } from '~hooks/useAmountLessFee.ts';
+import { NotificationType, type Notification } from '~types/notifications.ts';
+import { formatText } from '~utils/intl.ts';
+
+import NotificationMessage from '../NotificationMessage.tsx';
+
+const displayName =
+  'common.Extensions.UserHub.partials.MultisigNotificationMessage';
+
+interface MultisigNotificationMessageProps {
+  action: ColonyActionFragment | null | undefined;
+  colony: NotificationColonyFragment | null | undefined;
+  creator: string;
+  loading: boolean;
+  notification: Notification;
+}
+
+const MSG = defineMessages({
+  finalized: {
+    id: `${displayName}.finalized`,
+    defaultMessage: 'Finalized:',
+  },
+  approved: {
+    id: `${displayName}.approved`,
+    defaultMessage: 'Approved:',
+  },
+  rejected: {
+    id: `${displayName}.rejected`,
+    defaultMessage: 'Rejected:',
+  },
+  unknownAction: {
+    id: `${displayName}.unknownAction`,
+    defaultMessage: 'A multisig action was updated',
+  },
+  unknownChange: {
+    id: `${displayName}.unknownChange`,
+    defaultMessage: 'Multisig action updated: ',
+  },
+});
+
+const MultisigNotificationMessage: FC<MultisigNotificationMessageProps> = ({
+  action,
+  colony,
+  creator,
+  loading,
+  notification,
+}) => {
+  const { notificationType } = notification.customAttributes || {};
+
+  const amountLessFee = useAmountLessFee(action?.amount, action?.networkFee);
+
+  const actionMetadataDescription = useMemo(() => {
+    if (!action || !colony) {
+      return formatText(MSG.unknownAction);
+    }
+
+    return formatText(
+      { id: 'action.title' },
+      getActionTitleValues({
+        actionData: { ...action, amount: amountLessFee },
+        colony: {
+          nativeToken: {
+            ...colony.nativeToken,
+          },
+          metadata: colony.metadata,
+        },
+      }),
+    );
+  }, [action, amountLessFee, colony]);
+
+  const Message = useMemo(() => {
+    if (!creator || !action || !notificationType) {
+      return formatText(MSG.unknownAction);
+    }
+
+    if (notificationType === NotificationType.MultiSigActionCreated) {
+      return (
+        <>
+          {action.metadata?.customTitle}: {actionMetadataDescription}
```

Comment:
Could we add something in here that makes it clear it is a multi-sig motion and not just a regular action? Currently the notification looks exactly the same whether it is a permissions action or a multi-sig motion (the custom title is the only difference here).

<img width="447" alt="Screenshot 2024-10-10 at 14 50 42" src="https://github.com/user-attachments/assets/c4d0a560-4399-4154-bc5d-9eaf27beffc6">
 
Maybe just sticking `Multi-sig motion created:` at the start or something. Might need checking with product.